tests/test_adapter.py

```diff
@@ -1,8 +1,10 @@
+# flake8: noqa
```

Comment:
todo: remove when https://github.com/arangoml/pyg-adapter/pull/11#discussion_r936215059 is handled